.changeset/config.json

```diff
@@ -9,7 +9,6 @@
     "@xstate/immer",
     "@xstate/graph",
     "@xstate/inspect",
-    "@xstate/solid",
```

Comment:
I think that before unignoring this we should get a better understanding of: https://github.com/statelyai/xstate/issues/3925